tests/test_TopicNaming.py

```diff
@@ -129,6 +129,6 @@ def test_topic_naming_alias(xmlfile, csc, topic):
             index = 1
         else:
             index = 2
-        assert re.match(r"^[a-z]([a-zA-Z0-9_]*$)", name.text.split('_')[index]) is not None, \
+        assert re.match(r"^[a-z]([a-zA-Z0-9_]*$)", name.text.split('_', index)[index]) is not None, \
```

Comment:
Please consider providing the argument name in split, to make it clearer what the index is doing as an argument, in other words:
```
... name.text.split('_', maxsplit=index)[index])`
```

Reply:
Yes I added that too now. As for the proper field types: I can guess based on the telemetry received from the MQTT server but I prefer to do all of this in one go once we receive the information from DatControl since there will be big changes then anyway.